pkgs/misc/emulators/dolphin-emu/primehack.nix

```
@@ -0,0 +1,161 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeDesktopItem
+, pkg-config
+, cmake
+, wrapQtAppsHook
+, qtbase
+, bluez
+, ffmpeg
+, libao
+, libGLU
+, libGL
+, pcre
+, gettext
+, libXrandr
+, libusb1
+, lzo
+, libpthreadstubs
+, libXext
+, libXxf86vm
+, libXinerama
+, libSM
+, libXdmcp
+, readline
+, openal
+, udev
+, libevdev
+, portaudio
+, curl
+, alsa-lib
+, miniupnpc
+, enet
+, mbedtls
+, soundtouch
+, sfml
+, fmt
+, vulkan-loader ? null
+, libpulseaudio ? null
```

```suggestion
, vulkan-loader
, libpulseaudio
```

those packages are under normal condition not null.